docs/operating-scylla/nodetool-commands/restore.rst

```diff
@@ -4,9 +4,16 @@ Nodetool restore
 
 **restore** - Load SSTables from a designated bucket in object store into a specified keyspace or table
 
-Note that status of restore can be checked for ``user_task_ttl`` seconds after the operation is done.
-You can set the ttl using :doc:`nodetool tasks user-ttl </operating-scylla/nodetool-commands/tasks/user-ttl>`.
-If ``--nowait`` flag is not set, the command relies on ``user_task_ttl`` internally.
+The status of a restore operation is retained for a period defined by ``user_task_ttl``,
+allowing you to query the status even after the operation completes.
```

Comment:
```suggestion
allowing you to query the status even after the operation is completed.
```
I _think_ it is better this way.

Reply:
i am inclined to using "operation completes".